core/src/main/resources/log4j4ibis.xml

```diff
@@ -10,15 +10,15 @@
 	<Appenders>
 		<Appender type="Console" name="stdout">
 			<Layout type="IbisPatternLayout">
-				<Pattern>%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%t] %TC %c{2} - %m%n</Pattern>
+				<Pattern>%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%t] %X{mid,cid} %c{2} - %m%n</Pattern>
```

Comment:
Waarom is dit nu X?

Reply:
omdat TC niet (meer?) bestaat

Reply:
omdat TC niks (meer?) is. Nu worden messageId en correlationId afgedrukt. Die lijken me nuttig voor de hoeveelheid tekst die het inneemt op de regel.